language/stdlib/modules/LibraSystem.move

```diff
@@ -485,16 +492,29 @@ module LibraSystem {
                initialize_validator_set, set_libra_system_config;
     }
 
-    // TODO (dd): every member of validator_set has a ValidatorConfig that is not "None".
-    // This also implies that every member has a validator role.
-    // TODO (dd): Times out.  I think it should not be hard, so don't know why.
     spec module {
-       let validators = spec_get_validators();
-       invariant [deactivated, global] forall i1 in 0..len(validators): ValidatorConfig::is_valid(validators[i1].addr);
-    }
 
+       /// Every validator has a published ValidatorConfig whose config option is "some"
+       /// (meaning of ValidatorConfig::is_valid).
+       /// I'm not sure why this is important, but add_validator goes to the trouble
```

Comment:
We need to be careful with this kind of comments and `TODO(dd)` comments in the code, as talked about in our meeting on Friday.